src/Administration/Resources/app/administration/src/module/sw-media/mixin/video-cover.mixin.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @sw-package discovery
+ */
+Shopware.Mixin.register('video-cover-mixin', {
+    inject: [
+        'mediaService',
+        'acl',
+    ],
+
+    data() {
+        return {
+            showCoverSelectionModal: false,
+        };
+    },
+
+    computed: {
+        isVideoMedia() {
+            return this.isVideo(this.item);
+        },
+
+        hasVideoCover() {
+            return this.getCoverMediaId(this.item) !== null;
+        },
+    },
+
+    methods: {
+        openCoverSelectionModal() {
+            if (!this.acl.can('media.editor')) {
+                return;
+            }
+
+            this.showCoverSelectionModal = true;
+        },
+
+        closeCoverSelectionModal() {
+            this.showCoverSelectionModal = false;
+        },
+
+        async onCoverSelectionChange(selection) {
+            const [media] = selection;
+            this.closeCoverSelectionModal();
+
+            if (!media || !this.isImage(media)) {
+                this.createNotificationError({
+                    message: this.$tc('global.sw-media-media-item.notification.coverSelectionInvalid.message'),
+                });
+
+                return;
+            }
+
+            await this.persistCoverMedia(media.id);
+        },
+
+        async persistCoverMedia(coverMediaId) {
+            if (!this.isVideoMedia || !this.item?.id) {
+                return;
+            }
+
+            this.item.isLoading = true;
+
+            try {
+                await this.mediaService.assignVideoCover(this.item.id, coverMediaId);
+
+                const snippetKey = coverMediaId
+                    ? 'global.sw-media-media-item.notification.coverSaveSuccess.message'
+                    : 'global.sw-media-media-item.notification.coverRemoveSuccess.message';
+
+                this.createNotificationSuccess({
+                    message: this.$tc(snippetKey),
+                });
+
+                Shopware.Utils.EventBus.emit('sw-media-library-item-updated', this.item.id);
+            } catch {
+                this.createNotificationError({
+                    message: this.$tc('global.sw-media-media-item.notification.coverSaveError.message'),
```

Comment:
```suggestion
                    message: this.$t('global.sw-media-media-item.notification.coverSaveError.message'),
```
I guess I do not need to tell you everytime 😄 Please try not to re-introduce $tc